### PR TITLE
Frontend web development html css js - Data Attributes in Js

### LINKED ISSUE #3287: Data Attributes in Js
### Description

I will be creating documentation and giving introduction to Data Attributes in JavaScript.
I will also cover reading , writing and updating data Attributes in Js.

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by photon149**
<hr>
## Description 📜

I have added a summary about Data Attributes in Javascript.Mentioned about their working , procedure as well as provided with a real world example of the same.
Fixes #3287
Name : Shiwansh Raj
Sr. No. 64

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by photon149** [comment]
Please review the same :)
@code-reaper08 @praveenscience @moulikagrawal

**Review by SauravMukherjee44 — APPROVED** [review]

**Comment by photon149** [comment]
Thanks , please count this as my contribution towards Hacktoberfest as well.

**Review by moulikagrawal — CHANGES_REQUESTED** [review]
all your content should be in a single readme.md file, don't create separate files

**Comment by photon149** [comment]
Done the changes @moulikagrawal .Please check.

File: Web_Development/Javascript/4.Programming Fundamentals/Data Attribute/ATTRIBUTES_JS.MD [hunk 1]
```diff
@@ -0,0 +1,149 @@
+  
+# Data Attributes in Javascript
+
+![image](https://user-images.githubusercontent.com/74582422/135737483-6fc2bfd3-cec6-443a-a6bf-841721ee5682.png)
```
Comment (moulikagrawal) [inline Web_Development/Javascript/4.Programming Fundamentals/Data Attribute/ATTRIBUTES_JS.MD:4]: resize this image, it is too big on preview
rename this file as readme.md

**Review by moulikagrawal — CHANGES_REQUESTED** [review]
rename file as readme.md
and resize the top picture

**Comment by photon149** [comment]
Done @moulikagrawal .Please check once again.Also , count this as my contribution to be part of hacktoberfest as well.

**Review by moulikagrawal — APPROVED** [review]

**Pull request merged**